uefi/src/allocator.rs

```diff
@@ -22,6 +23,9 @@ use crate::table::boot::{BootServices, MemoryType};
 /// exited by the host application yet.
 static mut BOOT_SERVICES: Option<NonNull<BootServices>> = None;
 
+/// The memory type used for pool memory allocations.
+static mut MEMORY_TYPE: MemoryType = MemoryType::LOADER_DATA;
```

Comment:
Hm, I'd like to have the `Once` type here but it is a nightly-only API so far (<https://doc.rust-lang.org/core/cell/struct.OnceCell.html>).

Could you at least add a comment that this value is supposed to be set once during initialization and never alted again?